go/mysql/gtid_set.go

```diff
@@ -40,6 +40,10 @@ type GTIDSet interface {
 	// Contains returns true if the set is a superset of another set.
 	Contains(GTIDSet) bool
 
+	// Subtract returns a new GTIDSet that is the difference between the receiver
+	// and the supplied GTIDSet.
+	Subtract(GTIDSet) GTIDSet
```

Comment:
Usually called `Difference` for sets

Reply:
I like keeping it as `Subtract` just to match the MySQL built-in function `gtid_subtract()`, but let me know if you feel strongly and I'm happy to change it.